spec/support/matchers/has_errors_matcher.rb

```diff
@@ -3,7 +3,7 @@
 RSpec::Matchers.define :have_errors do |expected|
 
   match do
-    actual.body.match(/Error/)
+    actual.body.match(/Error\:/)
```

Comment:
This was raising intermittent errors, since Lorem ipsum contains the word "error"